DNN Platform/Library/Services/Upgrade/Upgrade.cs

```diff
@@ -3928,6 +3930,19 @@ public static int AddPortal(XmlNode node, bool status, int indent)
             return portalId;
         }
 
+        /// -----------------------------------------------------------------------------
+        /// <summary>
+        ///   Obsolete, AddPortal manages the Installation of a new DotNetNuke Portal
+        /// </summary>
+        /// <remarks>
+        /// </remarks>
+        /// -----------------------------------------------------------------------------
+        [Obsolete("Deprecated in DNN 9.3.0. Use the overloaded one with the 'superUser' parameter instead.")]

```

Comment:
Can we get one small change to this please, it should read.

Deprecated in DNN 9.3.0, will be removed in 11.0.0. Use the overloaded method with the 'superUser' parameter instead.

I'll make sure to update the PR documentation on this.  We just want to note, when it was deprecated when it will be removed, and the workaround.